source/extensions/omni.isaac.orbit/omni/isaac/orbit/terrains/terrain_importer.py

```diff
@@ -343,3 +343,11 @@ def _compute_env_origins_grid(self, num_envs: int, env_spacing: float) -> torch.
         env_origins[:, 1] = env_spacing * yy.flatten()[:num_envs] - env_spacing * (num_cols - 1) / 2
         env_origins[:, 2] = 0.0
         return env_origins
+
+    """
+    Dummy Functions.
+    """
```

Comment:
Can you move this to the section above "Internal helpers"? We try to keep private methods in the end and operations before.

You can call it "Operations - Sampling" instead of "Dummy".

Reply:
I have moved the section above the "Internal helpers", and renamed it in [e05bf0](https://github.com/NVIDIA-Omniverse/orbit/pull/191/commits/e075bf0ecbb4655f38ad46d9d161ab729e09c558).